org.buhocms.BuhoCMS.yml

```
@@ -0,0 +1,54 @@
+app-id: org.buhocms.BuhoCMS
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: buhocms
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=home
+modules:
+  - name: BuhoCMS
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/BuhoCMS
+      - cp -r buhocms data lib /app/BuhoCMS
+      - chmod +x /app/BuhoCMS/buhocms
+      - mkdir -p /app/bin
+      - ln -s /app/BuhoCMS/buhocms /app/bin/buhocms
+      - install -D installers/Flatpak/buhocms.svg -t /app/share/icons/hicolor/scalable/apps
```

this is the wrong name for the icon.
```suggestion
      - install -D installers/Flatpak/buhocms.svg /app/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
```